site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -0,0 +1,139 @@
+
+In the example above where `https://support.chat.example` is embedded on `https://retail.example`, the top-level URL is `https://retail.example`.  
+
+The partition key in that case is `("https", "retail.example")`.  
+
+Likewise, a _request's partition key_ is the site of the top-level URL the browser is visiting at the start of a request. Browsers must only send a cookie with the `Partitioned` attribute in requests with the same partition key as that cookie.  
+
+Here's what the cookie key in the example above looks like before and after CHIPS.
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/ZZm3G3cCgziUK2eezdiu.png", alt="ALT_TEXT_HERE", width="800", height="204" %}
+
+
+**Before CHIPS**
+
+```text
+key=("support.chat.example")
+```
+
+**After CHIPS**
+
+```text
+key={("https", "retail.example"),
+     ("support.chat.example")}
+```
+
+#### Security design
+
+To encourage good security practices, CHIPS proposes that partitioned cookies must be bound to the hostname (and not the registrable domain), and only be set by and sent over secure protocols. To ensure this, partitioned cookies must use the ``__Host-`` prefix.
+
+The `__Host-` prefix requires that the cookie be set with `Secure` and `Path=/` and disallows the `Domain` attribute. Since `Domain` cookies can be shared between different third-party subdomains within a partition, disallowing it makes partitioned cookies as close to being [origin-bound](docs/privacy-sandbox/glossary/#origin) as possible; and aligns cookies closer to the [Same-Origin Policy](https://developer.mozilla.org/docs/Web/Security/Same-origin_policy).
+
+Example: 
+
+```text
+Set-Cookie: __Host-example=34d8g; SameSite=None; Secure; Path=/; Partitioned;
+```
+
+### Implementation status
+
+Chrome has published an [Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/_dJFNJpf91U) for CHIPS and is in the process of writing code for the feature. Follow the [Chrome Platform Status entry](https://chromestatus.com/feature/5179189105786880) for progress.
+
+CHIPS is already available behind flags in Chrome 99. Check out the testing instructions and demo on [chromium.org](https://www.chromium.org/updates/chips/). Updated testing and debugging details will be added once the origin trials starts.
+
+To try it out locally, turn on the `chrome://flags/#partitioned-cookies` flag in Chrome Canary or use the `--partitioned-cookies=true` command line flag.
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/Afo08gb3WxuNT77Bi6xc.png", alt="ALT_TEXT_HERE", width="779", height="329" %}
+
+### Engage and share feedback
+
++   **GitHub**: Read the [proposal](https://github.com/WICG/CHIPS), [raise questions and follow the discussion](https://github.com/WICG/CHIPS/issues).
++   **Developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
```

Comment:
```suggestion
+   **Developer support**: Ask questions and join discussions on the [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).

```

Reply:
It's best practice to leave a blank line at the end of markdown files.

Reply:
I think the linter used to catch that...something weird's going on with the linter lately :)